### ISSUE TITLE
Extend Documentation

### ISSUE THREAD

**Issue by bboynton97** [smore-inc/tinys3#60]
Please include more detail in your documentation. Specifically, you mention that tinys3 can be used to get files from S3, but it's not shown in your documentation anywhere. I had to dig through the code to find it. This a great library, but the documentation is really lacking.

**Comment by audiofeature** [comment]
I agree, I also was looking to get objects from S3 rather than upload. Its mentioned to be supported, but completely undocumented!

**Comment by audiofeature** [comment]
see https://github.com/smore-inc/tinys3/pull/58/commits/31f0a9d810d941786f275a179d9f948b309b33c2

**Comment by audiofeature** [comment]
does not work: I get:

HTTPError: 400 Client Error: Bad Request for url: http://mybucket.s3.eu-west-2.amazonaws.com/test.mp3

**Comment by bboynton97** [comment]
Hey wait! I forgot, I wrote that on my other GitHub account, haha. Why hasn't it been pulled yet??
